src/test/mir-opt/match_false_edges/rustc.full_tested_match.PromoteTemps.after.mir

```diff
@@ -26,7 +26,7 @@ fn full_tested_match() -> () {
     bb0: {
         StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/match_false_edges.rs:15:13: 19:6
         StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/match_false_edges.rs:15:19: 15:27
-        _2 = std::option::Option::<i32>::Some(const 42i32,); // bb0[2]: scope 0 at $DIR/match_false_edges.rs:15:19: 15:27
+        _2 = std::option::Option::<i32>::Some(const 42i32); // bb0[2]: scope 0 at $DIR/match_false_edges.rs:15:19: 15:27
```

Comment:
Elsewhere you got rid of that `const` prefix, why not also here?

Reply:
I never touched a `const` prefix, those are from `mir::Operand::Const`.

Reply:
Well but the `const` prefix got removed for `bool` in this PR.

Reply:
See [here](https://github.com/rust-lang/rust/pull/71232/files#diff-18f31b6f5f6688e8b0e3ec9b7f768707) for example.

Reply:
The -/+ on the leftmost are my changes, the -/+ to the right of them are literal in the `.diff` file.

Reply:
I don't know enough LaTeX to draw a nice square, so I'll resort to a table:

| | Before `ConstProp` | After `ConstProp` |
|-|-:|:-|
| **Before this PR** | `std::option::Option::<bool>::Some(const true,)` | `{transmute(0x01): std::option::Option<bool>}` |
| **After this PR** | `std::option::Option::<bool>::Some(const true)` | `std::option::Option::<bool>::Some(true)` |

Reply:
```rust
                                  ConstProp
Option::<bool>::Some(const true,)┌─────────┐{transmute(0x01): Option<bool>}
                                 │         │
                         This PR │         │
                                 │         │
 Option::<bool>::Some(const true)└─────────┘Option::<bool>::Some(true)
```

Reply:
heh, fancy unicode. :D

Indeed I got confused by the double-diff at https://github.com/rust-lang/rust/pull/71232/files#r410220213, sorry for that.